src/lolex-src.js

```diff
@@ -557,7 +557,11 @@ function createClock(start, loopLimit) {
 
     function updateHrTime(newNow) {
         clock.hrNow += (newNow - clock.now);
-    }
+    };
```

Comment:
CI is unhappy about this semi

Reply:
CI is right, it makes no sense to end a function statement with a semicolon :D